sql/snapshot.sql

```diff
@@ -0,0 +1,100 @@
+-- public.global_snapshot definition
+
+-- Drop table
+
+-- DROP TABLE global_snapshot;
+
+CREATE TABLE global_snapshot (
+	ordinal int8 NOT NULL,
+	hash text NOT NULL,
+	height int8 NOT NULL,
+	subheight int8 NOT NULL,
+	last_snapshot_hash text NULL,
+	created_at timestamp NULL,
+	updated_at timestamp NULL,
+	CONSTRAINT global_snapshot_pkey PRIMARY KEY (ordinal)
+);
+CREATE INDEX global_snapshot_hash_idx ON public.global_snapshot USING btree (hash);
+
+
+-- public.dag_balance_change definition
+
+-- Drop table
+
+-- DROP TABLE dag_balance_change;
+
+CREATE TABLE dag_balance_change (
+	ordinal int8 NOT NULL,
+	address text NOT NULL,
+	balance int8 NULL,
+	CONSTRAINT dag_balance_change_pk PRIMARY KEY (ordinal, address),
+	CONSTRAINT dag_balance_change_global_snapshot_fk FOREIGN KEY (ordinal) REFERENCES global_snapshot(ordinal)
+);
+
+
+-- public.dag_transaction definition
+
+-- Drop table
+
+-- DROP TABLE dag_transaction;
+
+CREATE TABLE dag_transaction (
+	hash text NOT NULL,
+	global_ordinal int8 NULL,
+	block text NULL,
+	"source" text NOT NULL,
+	destination text NOT NULL,
+	amount int8 NOT NULL,
+	fee int8 NOT NULL,
+	block_hash text NOT NULL,
+	transaction_original_id int4 NULL,
+	created_at timestamp NULL,
+	updated_at timestamp NULL,
+	CONSTRAINT dag_transaction_pkey PRIMARY KEY (hash),
+	CONSTRAINT dag_transaction_global_snapshot_fk FOREIGN KEY (global_ordinal) REFERENCES global_snapshot(ordinal)
+);
+
+
+-- public.metagraph_snapshot definition
+
+-- Drop table
+
+-- DROP TABLE metagraph_snapshot;
+
+CREATE TABLE metagraph_snapshot (
+	metagraph_id text NOT NULL,
+	ordinal int8 NOT NULL,
+	global_ordinal int8 NULL,
+	hash text NOT NULL,
+	height int8 NOT NULL,
+	subheight int8 NOT NULL,
+	last_snapshot_hash text NULL,
+	created_at timestamp NULL,
+	updated_at timestamp NULL,
+	CONSTRAINT metagraph_snapshot_pk PRIMARY KEY (metagraph_id, ordinal),
+	CONSTRAINT metagraph_snapshot_global_snapshot_fk FOREIGN KEY (ordinal) REFERENCES global_snapshot(ordinal)
+);
+
+
+-- public.metagraph_transaction definition
+
+-- Drop table
+
+-- DROP TABLE metagraph_transaction;
+
+CREATE TABLE metagraph_transaction (
```

Comment:
Notes: 
- `hash` VARCHAR
- `metagraph_id` VARCHAR
- `block` - what is this?
- `source` / `destination` - same as above about addresses
- `transaction_original_id` - same as above
- `created_at` / `updated_at` - same as above